participant/styles/_uswds-theme.scss

```diff
@@ -16,5 +16,6 @@ in the form $setting: value,
   $theme-font-path: '/uswds/fonts',
   $theme-image-path: '/uswds/img',
   $theme-type-scale-lg: 7,
-  $theme-header-logo-text-width: 100%
+  $theme-header-logo-text-width: 100%,
```

Comment:
I don't have this pulled up on my local environment so maybe this isn't accurate, but using the Chromatic Storybook "small mobile" view, the header doesn't look as expected. I'd expect the blue background to extend the entire width. 
<img width="813" alt="image" src="https://user-images.githubusercontent.com/33822068/227994334-b6430698-dbd6-4314-a4d2-ffaf2dffdd69.png">

Reply:
I'm glad you are checking this! This is an aspect of using Chromatic/storybook. On the dev environment, it looks like this:

<img width="522" alt="Screen Shot 2023-03-27 at 12 07 06 PM" src="https://user-images.githubusercontent.com/67701/228042013-b517e21d-1236-4c0f-898e-221607cf3ace.png">